src/common/crypto/platform/openssl/crypto.cpp

```diff
@@ -600,15 +656,7 @@ expected::ExpectedString Sign(const Args &args, const mender::sha::SHA &shasum)
 }
 
 expected::ExpectedString SignRawData(const Args &args, const vector<uint8_t> &raw_data) {
```

Comment:
This function seems redundant now, it just calls the inner function with the same arguments. Can it be removed?

Reply:
Yes it can. 

I seem to recall we had a discussion about this naming with @lluiscampos when it was first introduced.

Which name do you prefer?

Reply:
I'd call it just `Sign`, since we sign both raw data and digest data using this function, depending on which key type it is.

Reply:
Removing it makes sense to me :+1: